asana/resources/gen/__init__.py

```diff
@@ -1 +0,0 @@
-# This file is automatically generated by generate.py using api.json
```

Comment:
This is a problem. We will need to preserve this file when deploying to the python library.